### DISCUSSION TITLE
Omnivore

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#70] [Announcements]
### Dev Blog
[I suspect Bedrock RTX is dead](https://discord.com/channels/846064688491528193/892517779612250182/1021631717880119307) so I am currently paying more attention to the Java and Mineways versions of my resource pack. I highly encourage you to try JG RTX with Mineways and Omniverse Create. So far SEUS seems to be the best shader for the Java version. Here are a few screenshots to compare the Java, Bedrock and Mineways versions:

> ![JG RTX in Omniverse Create](https://user-images.githubusercontent.com/1903667/198894527-0234ac90-7859-4724-b9c7-aafabf1514ab.png)
> Omniverse Create

> ![PBR JG with SEUS shader](https://user-images.githubusercontent.com/1903667/198894566-a91cdb63-6fdc-4db6-ae8a-6bb687dc26b1.png)
> SEUS

> ![PBR JG with BSL shader)](https://user-images.githubusercontent.com/1903667/198894957-fcf18a96-a77c-4851-8aae-bd0e1da9f855.png)
> BSL

> ![JG RTX in Bedrock with ray tracing enabled](https://user-images.githubusercontent.com/1903667/198894619-257e20a3-0a91-47cc-b071-414ddae9c636.png)
> Bedrock RTX

---

> ![JG RTX in Omniverse Create](https://user-images.githubusercontent.com/1903667/198895105-999a1403-f126-4929-b6ea-25e235e24f2d.png)
> Omniverse Create

> ![PBR JG in SEUS)](https://user-images.githubusercontent.com/1903667/198895145-078ae999-4f0b-42ee-857a-f5ec6328626d.png)
> SEUS

> ![PBR JG in BSL](https://user-images.githubusercontent.com/1903667/198895025-5c467d9a-6e28-4216-9c8d-f10b878a13d4.png)
> BSL

---

> ![JG RTX in Bedrock](https://user-images.githubusercontent.com/1903667/198894856-58e3479f-26a1-4ebc-9871-0fbeda7b9358.png)
> Bedrock RTX

> ![PBR JG in Java with SEUS](https://user-images.githubusercontent.com/1903667/198894881-35d5df41-4b9e-4806-a4bd-f13128676e78.png)
> SEUS

---

### Changelog

![New prismarine brick design in JG RTX](https://user-images.githubusercontent.com/1903667/198894374-fb3ae828-cb42-45b2-a3fa-f360efa3f99a.png)

#### Added
- (Bedrock) Bamboo sapling

#### Updated
- (Bedrock) Updated coral textures
- (Bedrock) Updated dead bush texture
- (Bedrock) New slushy ice design
- New prismarine brick design
- New sandstone designs
- Smaller oxeye daisy design
- New leather item frame background

#### Fixed:
- (Java) Clear glass texture is no longer opaque
- (Bedrock) Fixed missing iron block inventory texture
- (Bedrock) Removed metalness from water texture
- (Mineways) Included kelp textures
- (Mineways) Removed generated tiles

**Full Changelog**: https://github.com/jasonjgardner/jg-rtx/compare/v0.27.0...v0.28.0

<hr /><em>This discussion was created from the release <a href='https://github.com/jasonjgardner/jg-rtx/releases/tag/v0.28.0'>Omnivore</a>.</em>

**Comment by Expack3** [comment]
I don't know what Discord server the Discord link goes to, and whatever it is, I'm not a part of it, so I have no idea what the message is. Would it be possible to reproduce it here or elsewhere?

**Reply by jasonjgardner** [reply]
Oh duh! That was a careless mistake. Thanks for the heads up. I updated the release notes. Here's the gist:

> I think Mojang has moved on from ray tracing. Bundling Java and Bedrock now seems telling (in addition to the lack of updates/fixes). I get the impression: Java is for enthusiasts, who care about graphics and will take the time to find packs and shaders; whereas Bedrock is focused on add-ons and cross-platform.
